src/tests/ftest/list_tests.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python2 -u
```

Comment:
This (entire file) should not be necessary any more.  The CI nodes where launch.py is run to get tests lists should now have python3 and avocado 69.2.

It would be useful if you can remove this and use that instead so that I can get feedback on that packaging sooner rather than later.